moveit_core/collision_detection_bullet/CMakeLists.txt

```diff
@@ -25,29 +25,27 @@ target_link_libraries(${MOVEIT_LIB_NAME}
   ${BULLET_LIBRARIES}
 )
 
-add_library(collision_detector_bullet_plugin src/collision_detector_bullet_plugin_loader.cpp)
+add_library(collision_detector_bullet_plugin SHARED src/collision_detector_bullet_plugin_loader.cpp)
 set_target_properties(collision_detector_bullet_plugin PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
-ament_target_dependencies(collision_detector_bullet_plugin
+target_link_libraries(collision_detector_bullet_plugin PUBLIC
+  ${MOVEIT_LIB_NAME}
+  moveit_planning_scene
+)
+ament_target_dependencies(collision_detector_bullet_plugin PUBLIC
   rclcpp
   urdf
   visualization_msgs
   pluginlib
   rmw_implementation
 )
-target_link_libraries(collision_detector_bullet_plugin
-  ${MOVEIT_LIB_NAME}
-  moveit_planning_scene
-)
 
+install(DIRECTORY include/ DESTINATION include)
 install(TARGETS ${MOVEIT_LIB_NAME} EXPORT ${MOVEIT_LIB_NAME}
   TARGETS collision_detector_bullet_plugin EXPORT collision_detector_bullet_plugin
   LIBRARY DESTINATION lib
   ARCHIVE DESTINATION lib
   RUNTIME DESTINATION bin
```

Comment:
I think the runtime destination should be `lib/${PROJECT_NAME}`, otherwise all executables will be accessible in the global bin path. That's a change that we should probably apply for the whole codebase.

Reply:
OK, I can change this